systemvm/patches/debian/config/etc/init.d/cloud-early-config

```diff
@@ -1302,6 +1303,18 @@ EOF
   cp -f /etc/iptables/rt_tables_init /etc/iproute2/rt_tables
 }
 
+import_jvm_cacerts(){
+ log_it "importing jvm keystore to realhostip keystore"
+ keyStore=/usr/local/cloud/systemvm/certs/realhostip.keystore
+ storepass="vmops.com"
+ java_home=$(readlink -f $(which java) | sed "s:/bin/java::")
```

Comment:
not sure how systemvmtemplate-4.9 works.

in systemvmtemplate 4.6,
java_home=/usr/lib/jvm/java-7-openjdk-amd64/jre
java_keystore=/usr/lib/jvm/java-7-openjdk-amd64/jre/jre/lib/security/cacerts
two jre in java_keystore, which is wrong.

Reply:
I was using systemvmtemplate-4.10 from following link.

http://download.cloudstack.org/systemvm/4.10/

Reply:
in 4.10 i found the following path /usr/lib/jvm/zulu-8-amd64/jre/lib/security/cacerts

where is java_home defined?

Reply:
JAVA_HOMe is defined in cloud-agent script https://github.com/apache/cloudstack/blob/master/agent/distro/centos/SYSCONFDIR/rc.d/init.d/cloud-agent.in#L54

Reply:
@svenvogel java_home is a local variable that i have defined in import_jvm_cacerts.  

@karuturi cloud-agent script contains a code snippet as "export JAVA_HOME", but I don't think it initializes JAVA_HOME environment variable.